synapse/lib/stormlib/graph.py

```diff
@@ -14,6 +14,8 @@
         'scope': {'type': 'string', 'enum': ['user', 'power-up']},
         'creator': {'type': 'string', 'pattern': s_config.re_iden},
         'power-up': {'type': 'string', 'minLength': 1},
+        'maxsize': {'type': 'number', 'minimum': 0},
```

Comment:
Should this have a `default` value which matches the ast value of `100000` ?

If that is the case we'll need a migration for any stored graph definitions to add it?

Reply:
no need